common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -404,4 +407,17 @@ private String lookupRecursively(final String base, Map<String, String> found) {
     }
     return resolved;
   }
+
+  /**
+   * Constants.USER_FILE_BUFFER_BYTES shuold not bigger than Integer.MAX_VALUE bytes.
+   * @throws IllegalArgumentException if USER_FILE_BUFFER_BYTES bigger than Integer.MAX_VALUE
+   */
+  private void checkUserFileBufferBytes() {
+    if (!this.containsKey(Constants.USER_FILE_BUFFER_BYTES)) { //load from hadoop conf
+      return;
+    }
+    long usrFileBufferBytes =  getBytes(Constants.USER_FILE_BUFFER_BYTES);
+    Preconditions.checkArgument(((usrFileBufferBytes & Integer.MAX_VALUE) == usrFileBufferBytes),
+            "Invalid \"" + Constants.USER_FILE_BUFFER_BYTES + "\": " + usrFileBufferBytes);
```

Comment:
NIT: indentation doesn't look consistent with Tachyon code convention.
we indent 4 spaces for linebreak with the statement.